packages/core/tests/parameter/flag/formatting.spec.ts

```diff
@@ -728,5 +728,30 @@ describe("formatDocumentationForFlagParameters", function () {
             // THEN
             compareToBaseline(this, StringArrayBaselineFormat, lines);
         });
+
+        it("multipart flag", function () {
```

Comment:
unfortunately this test doesn't fail if i back my change out (it should show that `--multi.part` is incorrectly being interpreted as a positional param). I think this is an issue with how the tests are set up but I'm not familiar enough with the configuration to know what to change just now